TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -494,45 +530,32 @@ private static String formatAuthArgFlagsAsString() {
             ", ",
             REMOVE_AUTH_HEADER_VALUE_ARG,
             AUTH_HEADER_VALUE_ARG,
-            AWS_AUTH_HEADER_USER_AND_SECRET_ARG,
-            SIGV_4_AUTH_HEADER_SERVICE_REGION_ARG
+            SIGV_4_AUTH_HEADER_SERVICE_REGION_ARG,
+            "--target-username and --target-password"
+
         );
     }
 
     private static IAuthTransformerFactory buildAuthTransformerFactory(Parameters params) {
-        if (params.removeAuthHeader
-            && params.authHeaderValue != null
-            && params.useSigV4ServiceAndRegion != null
-            && params.awsAuthHeaderUserAndSecret != null) {
+        long authOptionsSpecified = Stream.of(
+            params.removeAuthHeader,
+            params.authHeaderValue != null,
+            params.useSigV4ServiceAndRegion != null,
+            params.targetUsername != null || params.targetPassword != null
+        ).filter(b -> b).count();
```

Comment:
nit - there's an Function.identity that you can use instead.  FWIW, I am partial the 4 characters that you've used :).